lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -181,7 +180,19 @@ func (ic *ICache) getImageByNamed(named reference.Named) (*metadata.ImageConfig,
 		}
 	}
 
-	return copyImageConfig(config), nil
+	return copyImageConfig(config)
+}
+
+// Don't assume the image id in image has "sha256:<id> as format.  We store it in
+// this format to make it easier to lookup by digest
+func prefixImageID(imageID string) string {
+	// Prefix was just "sha" which seems weird to be. Following general logic,
```

Comment:
`// Add the "sha256:" prefix to the image ID if missing`

would be more suitable.